.github/actions/build_extensions/action.yml

```diff
@@ -27,6 +27,9 @@ inputs:
     default: ''
 
   # Build config
+  duckdb_arch:
```

Comment:
could this be the same param as `deploy_as`? It should always be the same value, I think?

Reply:
Yes and no, they should always the same if they are both provided, but there can be cases where either `duckdb_arch` is not provided since it can be autodetected (say we are not cross-compiling) or cases where `deploy_as` is not provided since it doubles as "should it be also deployed".

A next step would be swapping `deploy_as` with a `also_deploy` boolean (that default to false) to avoid duplications / risk of them drifting apart, but I would prefer first to land it as is and double check stuff is working properly.